GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1242,6 +1242,13 @@ private void OnGridViewKeyUp(object sender, KeyEventArgs e)
                 return;
             }
 
+            // Check modifier detail if pressed.
+
+            if (e.Modifiers == Keys.None)
```

Comment:
This check is incorrect, should be !=
DEL/F2 only works with modifier now

Reply:
Done